lib/src/components/NxSegmentedButton/NxSegmentedButton.scss

```diff
@@ -52,49 +53,111 @@
 }
 
 .nx-btn--primary:not([disabled]) {
-  &.nx-segmented-btn__main-btn {
-    border-right-color: var(--nx-swatch-blue-60);
+  &.nx-segmented-btn__main-btn, &.nx-segmented-btn__dropdown-btn {
+    border-color: var(--nx-color-primary-segmented-button-divider);
+    position: relative;
+
+    &::after {
+      border: 2px solid transparent;
```

Comment:
Are all of the two's and four's in here interrelated? If so they should be based on a common variable.  Either a SCSS local variable, or if it makes sense (is this 2px the standard focus border width") as a documented CSS variable

Reply:
> Are all of the two's and four's in here interrelated? If so they should be based on a common variable. Either a SCSS local variable, or if it makes sense (is this 2px the standard focus border width") as a documented CSS variable

The 4px in the calc expression has no significance. It was a way to set the `outline-offset` to either -5px or -6px depending on the border width to keep the appearance that there was 2px of space between the border & outline. However, the 2px IS the standard focus border width, so I'll make a documented variable for that

Reply:
>>Are all of the two's and four's in here interrelated? If so they should be based on a common variable. Either a SCSS local variable, or if it makes sense (is this 2px the standard focus border width") as a documented CSS variable

>The 4px in the calc expression has no significance. It was a way to set the outline-offset to either -5px or -6px depending on the border width to keep the appearance that there was 2px of space between the border & outline. However, the 2px IS the standard focus border width, so I'll make a documented variable for that 👍

[aa61db3](https://github.com/sonatype/sonatype-react-shared-components/pull/1038/commits/aa61db3fbd2838c46c2c41b636b92ab76293240f)

Reply:
> [aa61db3](https://github.com/sonatype/sonatype-react-shared-components/pull/1038/commits/aa61db3fbd2838c46c2c41b636b92ab76293240f)

I'm guessing there are a bunch of other places beyond the scope of this ticket where that variable could/should be used.  If so we should put in a tech debt ticket to update them

Reply:
> I'm guessing there are a bunch of other places beyond the scope of this ticket where that variable could/should be used. If so we should put in a tech debt ticket to update them

I was thinking the same thing. Ticket to update: https://issues.sonatype.org/browse/RSC-1554